plugins/module_utils/network/common/utils.py

```diff
@@ -678,6 +678,10 @@ def validate_config(spec, data):
     basic._ANSIBLE_ARGS = to_bytes(json.dumps({"ANSIBLE_MODULE_ARGS": data}))
     validated_data = basic.AnsibleModule(spec).params
     basic._ANSIBLE_ARGS = params
+    if module:
+        module.no_log_values.update(
+            basic.list_no_log_values(spec, validated_data)
```

Comment:
welp, looks like this method got renamed in `devel` to `_list_no_log_values`

Reply:
Did you check if ti works with ansible 2.9, 2.10 as well?

Reply:
@ganeshrn Yes, this works with latest devel, 2.10, and 2.9.